schemas/com.snowplowanalytics.snowplow/currency_conversion_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,42 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for configuration of currency-conversion enrichment",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "currency_conversion_config",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+
+	"type": "object",
+	"properties": {
+		"vendor": {
+	        "type": "string"
+		},
+		"name": {
+	        "type": "string"
+		},		
+		"enabled": {
+	        "type": "boolean"
+		},
+		"parameters": {
+	        "type": "object",
+			"properties": {
+		        "apiKey": {
+				    "type": "string"
+				},
+				"baseCurrency": {
+			        "type": "string"
+				},
+				"rateAt": {
+				    "enum": ["EOD_PRIOR"]
+				}
+		    },         
+			"required": ["apiKey", "baseCurrency", "rateAt"],
+			"additionalProperties": false       
+		}
```

Comment:
Can you fix this ASAP @AALEKH - I want to do an Iglu Central release this afternoon...